kf_lib_data_ingest/etl/configuration/dataset_ingest_config.py

```diff
@@ -26,35 +31,48 @@ def __init__(self, config_path):
         """
         Construct a DatasetIngestConfig object from a config file
 
-        :param config_filepath: Path to the data ingest config file
+        :param config_path: Path to the data ingest config file
         """
-        # Is config path a dir or file
+        # If given config path is a dir, create default config file path
         config_path = os.path.abspath(os.path.expanduser(config_path))
         if os.path.isdir(config_path):
-            # If dir, create default config file path
             config_path = os.path.join(
                 config_path,
-                DATASET_INGEST_CONFIG_DEFAULT_FILENAME)
-
-        super().__init__(config_path, schema_path=DATA_INGEST_SCHEMA_PATH)
-
-        # Directory containing the extract config files
-        self.extract_config_dir = os.path.join(
-            os.path.dirname(self.config_filepath),
-            self.contents.get('extract_config_dir'))
-        self.extract_config_paths = [os.path.join(self.extract_config_dir,
-                                                  filename)
-                                     for filename in os.listdir(
-                                     self.extract_config_dir)
-                                     if filename.endswith('.py')]
+                DATASET_INGEST_CONFIG_DEFAULT_FILENAME
+            )
+
+        super().__init__(config_path)
+
+        # Validate
```

Comment:
This should all be inside the `_validate` method, which is part of the interface of the `base_config.py`. The `_validate` method gets called after the config file is read. See 
https://github.com/kids-first/kf-lib-data-ingest/blob/ae93b8ce2eabb25a1dc94aa74f6d9b4ef5ddd00b/kf_lib_data_ingest/etl/configuration/base_config.py#L26